src/constants/LocalStorageKeys.ts

```diff
@@ -0,0 +1,6 @@
+export const localStorageKeys = {
```

Comment:
the file should start with the small letter

Reply:
done